src/components/templates/authRequiredModal/index.js

```diff
@@ -1,3 +1,3 @@
-import AuthRequiredModal from './AuthRequiredModal';
+import AuthRequiredModal from "./AuthRequiredModal";
 
 export { AuthRequiredModal };
```

Comment:
export default

Reply:
closing this PR considering the time constraints, will cover this point in future branches